src/components/Globe.jsx

```diff
@@ -192,17 +193,20 @@ const MainSection = (props) => {
       </div>
 
       {props.showGlobe ? (
-        <ReactGlobe
-          height={'95vh'}
-          markers={markers}
-          options={options}
-          width="100%"
-          onGetGlobe={setGlobe}
-          globeCloudsTexture={null}
-          globeTexture={globeTexture}
-          globeBackgroundTexture={null}
-          initialCameraDistanceRadiusScale={3.5}
-        />
+        <>
+          <ReactGlobe
+            height={'95vh'}
+            markers={markers}
+            options={options}
+            width="100%"
+            onGetGlobe={setGlobe}
+            globeCloudsTexture={null}
+            globeTexture={globeTexture}
+            globeBackgroundTexture={null}
+            initialCameraDistanceRadiusScale={3.5}
+          />
+          <Timeline />
```

Comment:
I think `Timeline` should stay outside of the `Globe` and they should be independent from each other.

Reply:
Are you proposing that `Timeline` should be rendered outside of the `MainSection` component and in `App.js` instead? Like so:

      <ThemeProvider theme={theme}>
        <div className="App">
          <div className="globe-container">
            <div className="col-lg-12 col-sm-12">

              <MainSection
                showGlobe={this.state.showGlobe}
              />

            </div>
            <Timeline /> // insert Timeline component here? render based on state
            <div className="m-6">
              <div className="guide-1">
                <GroupedButton
                  showGlobe={this.state.showGlobe}
                  toggleDisplay={this.toggleDisplay}
                />
              </div>
            </div>
          </div>
          <Footer />
        </div>
      </ThemeProvider>

In this case, would we use `Context` to store the data retrieved in `Globe` so that `Timeline` is able to access the data?